nifi-commons/nifi-sensitive-property-provider/src/main/java/org/apache/nifi/properties/PropertyProtectionScheme.java

```diff
@@ -26,6 +26,7 @@
 public enum PropertyProtectionScheme {
     AES_GCM("aes/gcm/(128|192|256)", "aes/gcm/%s", "AES Sensitive Property Provider", true),
     AWS_KMS("aws/kms", "aws/kms", "AWS KMS Sensitive Property Provider", false),
+    AZURE_KEYVAULT("azure/keyvault", "azure/keyvault", "Azure Key Vault Sensitive Property Provider", false),
```

Comment:
As above, this would become AZURE_KEYVAULT_KEY and azure/keyvault/key

Reply:
Will make the changes